types/cytoscape/index.d.ts

```diff
@@ -3897,6 +3897,29 @@ declare namespace cytoscape {
              * A value between [0 1].
              */
             "border-opacity"?: PropertyValueNode<number> | undefined;
+            /**
+             * The position of the node’s border.
+             * One of: center, inside, outside.
+             */
+            "border-position"?: PropertyValueNode<"center" | "inside" | "outside"> | undefined;
+            /**
+             * The cap style of the node’s border.
+             * One of: butt, round, square.
+             */
+            "border-cap"?: PropertyValueNode<"butt" | "round" | "square"> | undefined;
+            /**
+             * The join style of the node’s border.
+             * One of: miter, bevel, round.
+             */
+            "border-join"?: PropertyValueNode<"miter" | "bevel" | "round"> | undefined;
+            /**
+             * The dashed line pattern which specifies alternating lengths of lines and gaps. (e.g. [6, 3]).
+             */
+            "border-dash-pattern"?: PropertyValueNode<number[]> | undefined;
```

Comment:
Can "border-dash-pattern" only accept lists of length 2? (if so, you should use a tuple)

Reply:
It accepts arrays of any length, the value is forwarded to [`CanvasRenderingContext2D.setLineDash`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash).

Reply:
Great, thanks!